src/main/java/com/github/dockerjava/api/command/CreateContainerCmd.java

```diff
@@ -417,6 +417,8 @@
 
     CreateContainerCmd withTty(Boolean tty);
 
+    CreateContainerCmd withTmpFs(Map<String, String> tmpFs);
```

Comment:
> you expect TmpFs to be only accessible with an HostConfig instance ?

yes

>there's a lot of with* methods like this one, right ?

Can't be removed atm because of backward compabitility